corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -107,4 +110,28 @@ FormplayerFrontend.on("start", function (options) {
             FormplayerFrontend.trigger("apps:list", options.apps);
         }
     }
+});
+
+FormplayerFrontend.on("sync", function () {
+    var user = FormplayerFrontend.request('currentUser');
+    var username = user.username;
+    var domain = user.domain;
+    var formplayer_url = user.formplayer_url;
+    var resp = $.ajax({
+        url: formplayer_url + "/sync-db",
```

Comment:
@benrudolph I'm not sure the best way to unify this with the Utils method since this function and the Backbone collections make this call in slightly different formats